pytrinamic/modules/TMCM6214.py

```diff
@@ -0,0 +1,256 @@
+            PowerDownDelay                   = 214
+            DeviationAction                  = 240
+            ReverseShaft                     = 251
+            UnitMode                         = 255
+
+        class ENUM:
+            """
+            Constant enums for parameters of this module.
+            """
+            MicrostepResolutionFullstep      = 0
+            MicrostepResolutionHalfstep      = 1
+            MicrostepResolution4Microsteps   = 2
+            MicrostepResolution8Microsteps   = 3
+            MicrostepResolution16Microsteps  = 4
+            MicrostepResolution32Microsteps  = 5
+            MicrostepResolution64Microsteps  = 6
+            MicrostepResolution128Microsteps = 7
+            MicrostepResolution256Microsteps = 8
+
+    class GP0:
+        """
+        Global parameter map for this module.
+        """
+        RS485Baudrate                 = 65
+        SerialAddress                 = 66
+        SerialHeartbeat               = 68
+        CANBitrate                    = 69
+        CANSendId                     = 70
+        CANReceiveId                  = 71
+        CANSecondaryId                = 72 #not in datasheet
+        TelegramPauseTime             = 75
+        SerialHostAddress             = 76
+        AutoStartMode                 = 77
+        ProtectionMode                = 81
+        CANHeartbeat                  = 82
+        CANSecondaryAddress           = 83
+        EepromCoordinateStore         = 84
+        ZeroUserVariables             = 85
+        SerialSecondaryAddress        = 86
+        ApplicationStatus             = 128
+        DownloadMode                  = 129
+        ProgramCounter                = 130
+        TickTimer                     = 132
+        RandomNumber                  = 133
+        SuppressReply                 = 255
+
+    class GP3:
+        Timer_0                        = 0
+        Timer_1                        = 1
+        Timer_2                        = 2
+        StopLeft_0                     = 27
+        StopRight_0                    = 28
+        StopLeft_1                     = 29
+        StopRight_1                    = 30
+        StopLeft_2                     = 31
+        StopRight_2                    = 32
+        StopLeft_3                     = 33
+        StopRight_3                    = 34
+        StopLeft_4                     = 35
+        StopRight_4                    = 36
+        StopLeft_5                     = 37
+        StopRight_5                    = 38
+        Input_0                        = 39
+        Input_1                        = 40
+        Input_2                        = 41
+        Input_3                        = 42
+        Input_4                        = 43
+        Input_5                        = 44
+        Input_6                        = 45
+        Input_7                        = 46
+
+
+    class IO:
+        IN0 = 0
+        IN1 = 1
+        IN2 = 2
```

Comment:
I think this module also hat IN4 to IN7, this class is used for the SIO command I guess.